### PR TITLE
1. wait for init timeout added, 2. gce api moved to the separated file

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

**Comment by amoskong** [comment]
Looks good to me.

File: sdcm/cluster_aws.py [hunk 1]
```diff
@@ -405,49 +405,12 @@ def _node_setup(self, node):
             time.sleep(5)
             node.remoter.run('nodetool status', verbose=True)
 
-    def wait_for_init(self, node_list=None, verbose=False):
```
Comment (amoskong) [inline sdcm/cluster_aws.py]: Can we set a big default timeout to avoid the stuck in special case?
Such as 120 mins?

@roydahan

Comment (roydahan) [reply sdcm/cluster_aws.py]: For now we handle it only for specific case, add node after decommission.
Let's see if we have other cases were the test is stuck before we starts to add more timeouts.

**Comment by roydahan** [comment]
@larisau is testing the code right now.
If it works, she will give her OK here and merge it.

Then, we'll need to BACKPORT it to branch-2.1 - @amoskong

**Comment by amoskong** [comment]
Backported 6 commits to branch-2.1

```
commit dc36cb585a9226f76d4a9866d98e60347ed60b02
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Mon Nov 20 18:27:21 2017 +0200

    verify stress thread: write warning to log if no results received
    
    (cherry picked from commit 42d0ab5b2159ae764aca17a124d7b6c3b8c60f08)

commit adb5ba94ce96d2444d6cea746aa2812368cec9db
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Mon Nov 20 07:17:12 2017 +0200

    wait for node setup fix
    
    (cherry picked from commit 1bb44e387644cd5ae0cb62c146b08ccee0d96e09)

commit df2ceb344c86799721f82cc6f8de25ec2f637d0e
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Sun Nov 19 19:46:02 2017 +0200

    verify_stress_thread: support number of stress processes per loader
    
    (cherry picked from commit 2d892c45b88e2a813ba32486860a407191851186)

commit c3ca720533e91496d851ccb150225e84d0081b9d
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Sun Nov 19 15:37:50 2017 +0200

    verify_stress_thread changed to update stress results in ES
    
    (cherry picked from commit 1fda1622874c12fbda4c5ef6cbdf528b5e342ed5)

commit dbd66285e4ade4a4475d63713cc60ca5521d02e1
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Thu Nov 16 13:36:44 2017 +0200

    nemesis decommission: retry adding new node if failed on timeout
    
    (cherry picked from commit 707790728d4bfa52661304422247f547b5d13753)

commit 5dc2a77808b153af07963c6e977d31593be58830
Author: Larisa Ustalov <larisau@scylladb.com>
Date:   Thu Nov 16 11:17:36 2017 +0200

    1. wait for init timeout added
    2. gce api moved to the separated file
    
    (cherry picked from commit 39fc8c85029a161b9d2ec8df5069cb3c481adf48)

```

**Pull request merged**